### DISCUSSION TITLE
Nuxt 3.9.0 Dev server startup extremely slow

### DISCUSSION THREAD

**Discussion by kylegl** [nuxt/nuxt#24984] [General]
Since i upgraded to Nuxt 3.9.0 the dev server startup has been terribly slow. went from something like 14000ms to 65000ms. Anyone else experiencing this? If not anyone know how to debug it?

**Comment by TheAlexLichter** [comment]
Also in fresh projects?

**Comment by kylegl** [comment]
Nope fresh project takes 600ms for nitro to build. Looks like im doing some digging
